io_scene_halo/file_ass/__init__.py

```diff
@@ -430,6 +445,7 @@ def execute(self, context):
             parser.add_argument('-arg3', '--game_version', dest='game_version', type=str, default="halo2")
             parser.add_argument('-arg4', '--folder_structure', dest='folder_structure', action='store_true')
             parser.add_argument('-arg5', '--hidden_geo', dest='hidden_geo', action='store_true')
+            parser.add_argument('-arg5', '--nonrender_geo', dest='nonrender_geo', action='store_true')
```

Comment:
Not sure what to do here. Would shifting the arg numbers to make room for it break anything? Or should it just be moved to the end?

Reply:
Shift the arg numbers or add it to the end. No preference either way personally since I don't think anyone actually uses this feature. Just make sure the arg isn't a duplicate.